src/utils/tab.js

```diff
@@ -150,10 +150,13 @@ tabUtils.broadCastTo = (id, msg, isSiteInsideFrame) => {
  * @param  {String}  msg
  * @param  {Boolean} isSiteInsideFrame
  */
-tabUtils.sendMessage = (target, msg, isSiteInsideFrame) => {
+tabUtils.sendMessage = (target, msg, isSiteInsideFrame, name) => {
   let origin = tabUtils.config.origin || '*';
-
-  if (isSiteInsideFrame && target.ref[0]) {
+  if (isSiteInsideFrame && name) {
+    for (let i = 0; i < target.ref.length; i++) {
+      target.ref[i].postMessage(msg, origin);
+    }
+  } else if (isSiteInsideFrame && target.ref[0]) {
```

Comment:
I think we can remove this if we don't check the name in prev if...it will become generic now